tools/PrepareRelease/SyncMsiContent.cs

```diff
@@ -14,15 +14,18 @@ public class SyncMsiContent
         private const string ComponentGroupDirectoryTemplate = @"{{component_group_directory}}";
         private const string FileIdPrefixTemplate = @"{{file_id_prefix}}";
         private const string FrameworkMonikerTemplate = @"{{framework_moniker}}";
+        private const string Net461Property = "WIX_IS_NETFRAMEWORK_461_OR_LATER_INSTALLED";
+        private const string Net461Condition = "<Condition>" + Net461Property + "</Condition>";
```

Comment:
I'm fine with the output as is since we're only updating them programmatically, but I think we could solve the blank line problem with something like this:

```csharp
private const string Net461Condition = @"
      <Condition>" + Net461Property + "</Condition>";
```

and

```csharp
private static readonly string ItemTemplate = $@"
      <Component Win64=""$(var.Win64)"">{Net461Condition}
...
";
```

Reply:
Much better, thanks